examples/txDeposit.ts

```diff
@@ -0,0 +1,48 @@
+import { UserFactory } from "../libs/user";
+import { config } from "./appConfig";
+
+// Example script
+const main = async () => {
+  let user;
+  try {
+    // # 1 Create an instance of User (mnemonic is optional)
+    user = await UserFactory.create({
+      blockchainWsUrl: config.blockchainWsUrl,
+      clientApiUrl: config.clientApiUrl,
+      ethereumPrivateKey: config.ethereumPrivateKey,
+    });
+
+    // # 2 [OPTIONAL] If you did not pass a mnemonic, you can retrieve it
+    // COMING SOON
+
+    // # 3 [OPTIONAL] You can check API Client, blockchain ws connection
+    const status = await user.checkStatus();
+    console.log(status);
+
+    // # 4 Make deposit
+    const tokenAddress = config.tokenAddress;
+    const tokenStandard = "ERC20";
+    const value = "0.0001";
+    const txReceipts = await user.makeDeposit({
+      tokenAddress,
+      tokenStandard,
+      value,
+    });
+    console.log("Transaction receipts", txReceipts);
+
+    // # 5 [OPTIONAL] You can check the transaction hash
```

Comment:
I think you forgot some instruction here :)

Reply:
It's in L34 but perhaps will be clearer if we assign first to `const`, thank you!